packages/webview_flutter/webview_flutter_wkwebview/ios/Classes/FLTWFInstanceManager.m

```diff
@@ -0,0 +1,100 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import "FLTWFInstanceManager.h"
+
+@interface FLTThreadSafeMapTable<KeyType, ObjectType> : NSObject
++ (nonnull FLTThreadSafeMapTable *)strongToStrongMapTable;
```

Comment:
Nit: making this slightly different from the name in the class you are emulating (by removing `Objects`) seems needlessly confusing.